tests/e2e/onboarding.test.ts

```diff
@@ -131,6 +140,134 @@ test('onboarding with a short code', async ({ page, getOnboardingData }) => {
 	await expect(page).toHaveURL(`/onboarding`)
 })
 
+test('onboarding with GitHub OAuth', async ({ page }) => {
+	const code = `${MOCK_CODE_GITHUB}_${crypto.randomUUID()}_PLAYWRIGHT_TEST`
```

Comment:
If the intention is to create a random `code` for each test run, may I suggest [`testInfo.testId`](https://playwright.dev/docs/api/class-testinfo#test-info-test-id)? 

- Playwright already generates that;
- We get free collocation between the code and the test that generated it.

Reply:
I initially wanted the code to resemble the one we see, but I know realize that binding it ti the test instance makes much more sense. updated